packages/node-app/src/utils/inside.util.ts

```diff
@@ -0,0 +1,10 @@
+import { Requirements } from "../models/giveaway.model"
+
+// TODO: connect to inside database
+export const getParticipantAddress = (participant: string): string => {
+  return 'PARTICIPANT_ADDRESS';
+}
```

Comment:
We can send the email address directly in the request, since the login is with google auth